src/v2.rs

```diff
@@ -827,16 +837,16 @@ mod tests {
     let module = eszip.get_module("file:///main.ts").unwrap();
     assert_eq!(module.specifier, "file:///main.ts");
     let source = module.source().await;
-    assert_eq!(&*source, include_bytes!("./testdata/emit/main.ts"));
+    assert_matches_file!(source, "./testdata/emit/main.ts");
```

Comment:
These statements previously did not give good assertion errors. This is improved now. I used a macro instead of a function because then it says the line number for what assertion failed.